### ISSUE TITLE
Using Chrome: Scrolling up within the Channel/User list takes forever

### ISSUE THREAD

**Issue by MondaySam** [RocketChat/Rocket.Chat#22362]
Now this is a very strange bug I am experiences for a couple of weeks/ a few months now,  using Rocket.Chat within my Chrome browser.

I have a somewhat long list of channels and users in the left hand panel (maybe 50-70 users/channels)
Scrolling _down_ on this list is super quick and I usually can reach the end of the list with one or two turns of the mouse wheel.

However, when I try to scroll back up - using the mouse-wheel - it takes more than 10 turns of the wheel.

This only seems to be the case when using Chrome. Firefox and Edge are fine.

I also tried clearing cache data and what not. The bug still persists.

I am using Chrome 91.0.4472.106 (updated today) and Rocket.Chat Server 3.12

**Comment by ankar84** [comment]
Hi and thanks for reporting this.

Can you please use a bug template if you are reporting an issue - this information is extremely important for triage:

https://github.com/RocketChat/Rocket.Chat/issues/new?assignees=&labels=&template=bug_report.md

Make sure you follow the guidelines:

https://rocket.chat/docs/contributing/reporting-issues

That includes testing on the latest version of Rocket.Chat - you can test on open.rocket.chat in the sandbox channel.

Note also that for general questions and help you are better off asking in https://forums.rocket.chat or https://open.rocket.chat #support as github is reserved for bugs.

In most instances when doing this just hit escape to cancel. But that may depend on your version and client.

**Comment by emikolajczak** [comment]
Hi, as a workaround you can use Medium in View Mode

![image](https://user-images.githubusercontent.com/50440047/122036107-7a627b00-cdd3-11eb-9d9b-75db8c3a9cb1.png)

**Comment by MondaySam** [comment]
@ankar84 I will try to provide as many information as possible, but I am just a user and don't have access to the server/deploy information. Though I'm guessing that those aren't that relevant seeing that this is a frontend issue.

@emikolajczak Thanks, but changing the view mode doesn't change the issue at all for me. I usually run "condensed" and using a bigger view mode just makes scrolling up that more tedious. ;)

**Comment by ankar84** [comment]
> Should I make another ticket, or should I edit this one to include the missing information?

No, you can just edit you first post with all information you know.

In fact I confirm that strange behavior scrolling up in sidebar in Chrome.

**Comment by dcorrigangoempyrean** [comment]
I'm experiencing the same issue. Changing the view to Medium does not fix the issue. On Condensed mode, per "mouse wheel turn", it scrolls DOWN about 10 names and UP about 3.

**Comment by fleaz** [comment]
Same problem over here, **BUT** in  contrast to all the other people in this issue  I only have this behavior when  using the app. In all my browsers (FF and Chromium) this works perfectly fine.

**Comment by fleaz** [comment]
Dupe of #20994
